Updated Framework to 4.6.1, Added missing code file.

SDragon42 opened the pull request:
Changed the Target Framework of all projects to 4.6.1
Added the reference "VRage.Input.dll" to the Gilgame.SEWorkbench.Interop project.
The  file "BackupRequestedEventArgs.cs" is missing from the Gilgame.SEWorkbench project. Created a new one.
Removed the dependecy "SharpDX.Toolkit.Graphics.dll" as it doesn't exist
anymore.

Commit:
Updated Framework to 4.6.1, Added missing code file.

Changed the Target Framework of all projects to 4.6.1
The  file "BackupRequestedEventArgs.cs" is missing. Created a new one.
Removed the dependecy "SharpDX.Toolkit.Graphics.dll" as it doesn't exist
anymore.

MariuszPL55 commented:
I need that fix, I have problem with that what you describe: "SharpDX.Toolkit.Graphics.dll" ;/

Commit:
Engine.cs changes by Hippomormor (http://forum.keenswh.com/members/hippomormor.3975729/)

Commit:
Updated Revision for release

Commit:
Updated for SE 1.153

Added "VRage.Render.dll" to the assembly list.

Commit:
Removed missing dependency assembly.

Removed "SharpDX.Toolkit.dll" from the Dependency list.
Moved creation of the Dependency to a static constructor.

Commit:
Added nuget package "Blend.Interctivity.WPF".

Commit:
Changed path to Steam libraries to use the build variable.

Commit:
Replaced local Xceed AvalonDock project with nuget references.

Commit:
Added more nuget packages

Commit:
Excluded the nuget packages folder from git

Removed downloaded nuget packages from git storage

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into Changed2NugetProjects

Commit:
Revert "Added more nuget packages"

This reverts commit a9d08a995c4c1ff1d81af583aabbbb7a7eb5a440.

Commit:
Added a gitignore for nuget package binaries

Commit:
fixed ICSharpCode.NRefactory references

SDragon42 committed:
Merge pull request #1 from SDragon42/Changed2NugetProjects

Changed2nugetprojects

Commit:
Merge remote-tracking branch 'refs/remotes/gilgame/master'

Commit:
Revert "Merge remote-tracking branch 'refs/remotes/gilgame/master'"

This reverts commit c57980aa73fdbbd735a2c3fb0bb8379ef975521e, reversing
changes made to 8284f8c1a0fb7bf33fe38d12a7d450d1ee8d36e4.
